oem/ibm/host-bmc/host_lamp_test.cpp

```diff
@@ -70,6 +70,11 @@ bool HostLampTest::asserted(bool value)
                 "Get Effecter ID failed, effecter = 0");
         }
     }
+    else
```

Comment:
instead of a new else block here can we not set the DBus property to false in line 37 itself?

Reply:
line no 37 condition only hit if value=false and assertproperty=false.
but in lamp test assertproperty=true and we are trying to set value =false which will come to new Else block and make assert property to false.

Reply:
We enter line37 only if we are trying to set false value to the asserted property which is already false. In this case existing asserted property is true.